DesignSafe-CI/portal
# task/DES-2561: create Scratch dir on login

## Overview: ##

Creates a dir for the Frontera Scratch system upon login.

## PR Status: ##


## Related Jira tickets: ##

* [DES-2561](https://jira.tacc.utexas.edu/browse/DES-2561)

## Testing Steps: ##
1. Add `AGAVE_WORKING_SYSTEM=designsafe.storage.frontera.scratch` to `designsafe.env` (this is also set as default if no `AGAVE_WORKING_SYSTEM` is set
2. Login to portal
3. SSH into frontera
4. Confirm you have a directory at `/scratch1/00849/tg458981/designsafe-scratch/$username`

## Notes:
- The setting `AGAVE_WORKING_SYSTEM: designsafe.storage.frontera.scratch` will need to be added to the ansible playbook group_vars before deploy
